File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EErrorDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EErrorDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd41
            AppTheme.STANDARD -> R.color.cstd41
            AppTheme.BLUE -> R.color.cbd41
            AppTheme.GREEN -> R.color.cgd41
            AppTheme.RED -> R.color.crd41
            AppTheme.NOTHING -> R.color.cnod41
        }
    }
}

File: lib/layer2_core/dark/e_error_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EErrorDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd41;
      case 'standard': return cstd41;
      case 'blue': return cbd41;
      case 'green': return cgd41;
      case 'red': return crd41;
      case 'nothing': return cnod41;
      default: return cstd41;
    }
  }
}
